great_expectations/datasource/fluent/spark_file_path_datasource.py

```diff
@@ -569,6 +569,24 @@ def _get_reader_options_include(self) -> set[str]:
         return {"timestamp_as_of", "version_as_of"}
 
 
+class DirectoryDeltaAsset(_DirectoryDataAssetMixin, DeltaAsset):
+    type: Literal["directory_delta"] = "directory_delta"  # type: ignore[assignment]
+
+    @classmethod
+    def _get_reader_method(cls) -> str:
+        return "delta"
```

Comment:
❤️ @anthonyburdi This is really nice and informs other situations where the assets look like database type while being computed by Spark (DataFrame like).  Thank you!